cg/meta/workflow/mip_dna.py

```diff
@@ -73,3 +73,14 @@ def get_genome_build(self, case_id: str) -> str:
         """Return the reference genome build version of a MIP-DNA analysis."""
         analysis_metadata: MipAnalysis = self.get_latest_metadata(case_id)
         return analysis_metadata.genome_build
+
+    def get_data_analysis_type(self, case_id: str) -> str:
+        """Return the data analysis type of a MIP-DNA analysis
+        Patch for the typical behaviour of the AnalysisAPI function.
+        It does not raise an error with multiple analysis types.
+        """
+        case: Case = self.get_validated_case(case_id)
+        analysis_types: set[str] = {
+            link.sample.application_version.application.analysis_type for link in case.links
```

Comment:
I would still raise a warning and maybe set it to `WGS` if we have multiple types (that was the problem right? `WES` samples being part of a `WGS` run), but it would be good if someone with more knowdledge of MIP could confirm this

Reply:
@peterpru ?

Reply:
I agree with Vadym here. Usually it is a `WGS` affected together with `WES` parents. The affected will get full `WGS` output.

Reply:
Alright, then with multiple analysis types i will not return wgs